src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -63,7 +63,7 @@ public function exists($name)
      *
      * @throws \Twig_Error_Loader if the template could not be found
      */
-    protected function findTemplate($template)
+    protected function findTemplate($template, $throw = true)
     {
         $logicalName = (string) $template;
```

Comment:
- if the loader is the symfony one, it will still work AFAIK if we pass a string because that is what the parser is for: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php#L83
- if it's a twig loader, it will not understand the symfony notation but still work as expected since it's a string

Or am I wrong?